### PR TITLE
Update plan-for-government-dod.md

### PR DESCRIPTION
**Pull request by cowitkow**
Planned feature: https://domoreexp.visualstudio.com/MSTeams/_workitems/edit/460108

### CODE REVIEW

**Comment by LolaJacobsen** [comment]
@David-Rosenthal - please review

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/plan-for-government-dod.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ To accommodate the requirements of our government cloud customers, there are som
 | | Email a channel | Not available |
 | | Add member | Available |
 | | Guest access | On the Government backlog |
+| | Private channels | On the government backlog |
```
Comment (JohanFreelancer9) [inline Teams/plan-for-government-dod.md]:
```suggestion
| | Private channels | On the Government backlog |
```

**Comment by LolaJacobsen** [comment]
@cowitkow I'm closing this PR without merging, per @daro: "We'll be shortly redirecting that section to the Teams Service Description."

**Pull request closed without merging**